src/rules/require-sql-template.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+import assert from 'assert';
+import get from 'lodash.get';
+import parser from 'simple-sql-parser';
+
+/**
+ * Check if String is an SQL query.
+ */
+
+function isSQLQuery(literal) {
+  return literal && Object.keys(parser.sql2ast(literal)).length > 0;
+}
+
+/**
+ * Export `requireSqlTemplate`.
+ */
+
+const requireSqlTemplate = () => {};
+
+requireSqlTemplate.prototype = {
+  check: (file, errors) => {
+    file.iterateNodesByType('VariableDeclaration', (node) => {
+      node.declarations.forEach((declaration) => {
+        const name = get(declaration, 'init.tag.name');
+        const literal = get(declaration, 'init.value');
+
+        if (isSQLQuery(literal) && name !== 'sql') {
```

Comment:
Not sure if it helps here, since I would need to add an additional `if` statement.